File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int cons = 0;
        for(int i = 0;i<nums.length;i++){
            cons = nums[i] == 1 ? cons+1:0;
            max = Math.max(max,cons);
        }
        return max;
        
    }
}

File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let max = 0;
    let cons = 0;
    for(let dig of nums){
        if (dig === 1){
            cons++;
            max = Math.max(cons,max);
        }
        else
        cons = 0;
    }
    return max;
    
};